docs/rules/camelcase.md

```diff
@@ -1,6 +1,6 @@
-# Require Camelcase (camelcase)
+# Require CamelCase (camelcase)
```

Comment:
I don't think the canonical spelling of camelcase is camelCase.

Reply:
Should I change it back to `Camelcase` or should I use `Camel-case` or something else?

Reply:
I think the word is "camelcase" (or "Camelcase" when title-cased).

I don't feel as strongly about this one as I do the RegExps change. Feel free to wait for a second opinion on this one if you like.

Reply:
Wikipedia’s article is named [Camel case]9https://en.wikipedia.org/wiki/Camel_case), but it says:

> stylized as **camelCase** or **CamelCase**

Google’s [definition](https://encrypted.google.com/search?hl=en&q=spell%20camelcase) says:

> noun: **camelCase**

DuckDuckGo [says](https://duckduckgo.com/?q=spell+camelcase&t=osx&ia=answer):

> **Camelcase** does not appear to be spelled correctly.
> Suggestions: camel case, camel-case, camels, camellias, Camel's, camel's.

Reply:
Well, I'll be. Okay, I'll defer to other reviewers' consensus here.